Apps/Contoso.WPF.Demo/MainWindow.xaml

```diff
@@ -65,7 +65,7 @@
                         <StackPanel Orientation="Vertical">
                             <DockPanel LastChildFill="True">
                                 <Label Content="Event Name"/>
-                                <TextBox Name="EventName" VerticalAlignment="Center" />
+                                <TextBox Name="eventName" VerticalAlignment="Center" />
```

Comment:
this change in casing is not correct.

Reply:
@guperrot That's not a change itself, but coping the changes from puppet
https://github.com/microsoft/appcenter-sdk-dotnet/blob/f951794d2adea865c8d57bae9eaeba30228289d3/Apps/Contoso.WPF.Puppet/MainWindow.xaml#L68